src/python/grpcio/grpc/utilities.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2016, Google Inc.
```

Comment:
[We had a recent discussion about what effectively amounted to how one measures lesser evils.](https://github.com/grpc/grpc/pull/6327#discussion_r63936800) :-|

But, I guess I'm over it.